arviz/stats/stats.py

```diff
@@ -142,6 +150,11 @@ def compare(
     loo : Compute the Pareto Smoothed importance sampling Leave One Out cross-validation.
     waic : Compute the widely applicable information criterion.
 
+    Notes
+    -----
+    If the `log_likelihood` group is not present in the input datasets, ArviZ will attempt
+    to compute it.
```

Comment:
I don't think this is right nor possible even. If compare is working without the log likelihood group present it *must* be because log likelihood data is stored as `log_likelihood` variable in `sample_stats` group. This was the original approach and still works for backward compatibility with old netcdf files.